web/pdf_sidebar_resizer.js

```diff
@@ -34,7 +34,7 @@ class PDFSidebarResizer {
    * @param {IL10n} l10n - Localization service.
    */
   constructor(options, eventBus, l10n = NullL10n) {
-    this.enabled = false;
+    this.enabled = true;
```

Comment:
I would say we can get rid of `this.enabled` altogether. It's always `true` now and we don't have similar toggles in other components.

Reply:
> I would say we can get rid of `this.enabled` altogether. It's always `true` now and we don't have similar toggles in other components.

That's a good point; it's gone now :-)